### ISSUE TITLE
Abouit Gem layer and  ArcFaceloss 

### ISSUE THREAD

**Issue by ChengHuan2020** [Rohit102497/MABNet#2]
Dear Prof.Rohit Agarwal:
    In the paper,  **KL divergence loss** is calculated using the Gem layer embeddings, but the code version provided does not include gem pooling , and the line **arcface_loss = losses.ArcFaceLoss**  is missing. Could you please provide the complete code version?

**Comment by Rohit102497** [comment]
Hello @ChengHuan2020 ,

Please check line number 223 in the main_dino.py file.
